scene/3d/sprite_3d.cpp

```diff
@@ -770,10 +770,13 @@ void Sprite3D::_draw() {
 	}
 
 	Rect2 base_rect;
+	bool texture_repeat;
 	if (region) {
 		base_rect = region_rect;
+		texture_repeat = (region_rect.size.x > texture->get_width()) || (region_rect.size.y > texture->get_height());
```

Comment:
This check is incorrect, repeat should be true if the region anyhow exceeds the texture.

This should do:
```suggestion
		texture_repeat = !Rect2(Vector2(), tsize).encloses(region_rect.abs());
```
I've used `region_rect.abs()` there so regions with negative sizes would be handled according to the current behavior for them (`encloses` expects non-negative-size rects).

In v4.4.1.stable.official [49a5bc7b6]:
![qiSIIgzkdk](https://github.com/user-attachments/assets/8cacbd44-b2cb-433a-9658-4577e8856363)|![TeW8IyUXNi](https://github.com/user-attachments/assets/82ecb4c2-e9ee-4db0-a60d-f2a4b18285a3)
-|-